src/vs/workbench/api/common/extHost.protocol.ts

```diff
@@ -2417,6 +2417,7 @@ export interface ExtHostTerminalServiceShape {
 	$acceptTerminalMaximumDimensions(id: number, cols: number, rows: number): void;
 	$acceptTerminalInteraction(id: number): void;
 	$acceptTerminalSelection(id: number, selection: string | undefined): void;
+	$acceptTerminalShellType(id: number, shellType: string): Promise<TerminalShellType>;
```

Comment:
Should return `void` here

Reply:
Changed to void [here](https://github.com/microsoft/vscode/pull/237624/commits/73a79dde410e88305fda658981f4dadc0e97e932#diff-d0ad8aca1603902c96b672cf64c9bb8442eb1c6e434452d3e2fbf1bbef2bb135R2420)